crates/beerus-rpc/src/utils.rs

```diff
@@ -0,0 +1,11 @@
+use ethers::types::U256;
```

Comment:
I think that the utils file should go in the beerus-core utils folder: 
https://github.com/keep-starknet-strange/beerus/blob/main/crates/beerus-core/src/ethers_helper.rs

Reply:
It's a very generic function of formatting we may use it for other RPC purposes.

What if tomorrow some starknet output must be hexadecimal?

But if it's sound better to you, let me know and I'll move it to the place you point out.

Reply:
Or perhaps changing to a generic function or macro.